### PR TITLE
Core explainability

### PR DESCRIPTION
**Pull request by aminatkhamokova**

### CODE REVIEW

File: holisticai/explainability/metrics/core/all_metrics.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    conditional_features_importance_indexes: list[Iterable],
+):
+    """
+    Parameters
+    ----------
+    feature_importance_indexes: Iterable
+        array with feature importance indexes
+    conditional_feature_importance_indexes: list[Iterable]
+        array with conditional feature importance indexes
+    """
```
Comment (crismunoz) [inline holisticai/explainability/metrics/core/all_metrics.py:30]: Please add  (for Readthedocs) ad description of the output:

Return:
--------
?????

File: tests/explainability/test_all_metrics.py
Comment (crismunoz) [inline tests/explainability/test_all_metrics.py:1]: test_multiclass_classification_position_parity?

**Review by crismunoz — CHANGES_REQUESTED** [review]

File: tests/explainability/test_all_metrics.py
Comment (aminatkhamokova) [reply tests/explainability/test_all_metrics.py:1]: The original version does not include testing for a multiclass classification setting

File: holisticai/explainability/metrics/core/all_metrics.py (hunk 1 above)
Comment (aminatkhamokova) [reply holisticai/explainability/metrics/core/all_metrics.py:30]: added

File: tests/explainability/test_all_metrics.py
Comment (crismunoz) [reply tests/explainability/test_all_metrics.py:1]: you have added "multiclass_classification" in extractor_utils.py, if we are going to use that in prod we need to test it.

Comment (aminatkhamokova) [reply tests/explainability/test_all_metrics.py:1]: added

**Review by crismunoz — APPROVED** [review]
great!

**Pull request merged**